lib/routers/establishment/project-versions.js

```diff
@@ -112,16 +119,19 @@ router.put('/:id/:action',
   perms('project.update'),
   validateAction,
   canUpdate,
+  userCanUpdateVersion,
```

Comment:
Specifically here, which I _think_ is the endpoint to perform a `fork` operation and start an amendment.

Reply:
Ahhh no. Not here.

Nothing to see here...

Reply:
it _should_ be an endpoint on the project